plugins/hotkeyshortcuts.koplugin/defaults.lua

```diff
@@ -0,0 +1,117 @@
+-- typed on my Commodore 64 ;)
+local Device = require("device")
+
+return {
+    hotkeyshortcuts_fm = {
+        modifier_plus_up                 = nil,
+        modifier_plus_down               = nil, -- keep nil.
+        modifier_plus_left               = nil,
+        modifier_plus_right              = nil,
+        modifier_plus_left_page_back     = nil, -- keep nil.
```

Comment:
What's the difference between lines with `= nil,` and those with `= nil, -- keep nil.`.
If there is one, write about it so one newcomer like me can get it. If none, remove it so I won't ask the question again :)

Reply:
My reading would be something like "already assigned outside the plugin." Which if correct is probably a better comment. ^_^

Reply:
that is exactly the case, i was originally going to write it differently but apparently never got around it.

Reply:
the difference is, one is nil because I don't have or want to assign anything to it atm. the other other one is nil with `-- keep nil` to indicate that nothing should ever be assigned to it. More importantly though, there is no real reason to touch those default values ever again...